docs/2_2_common_mechanisms.adoc

```diff
@@ -369,6 +369,10 @@ Exceptions:
 
 All strings passed as arguments to `fmi3SetString`, as well as all binary values passed as arguments to `fmi3SetBinary`, must be copied during these function calls, because there is no guarantee of the lifetime of strings or binary values, when these functions return.
 
+_[Note: In Scheduled Execution, Clocks are neither activated nor deactivated by the importer using <<fmi3SetClock>>._
+_Instead: The activation of a clock requires the importer to call <<fmi3ActivateModelPartition>>._
+_For triggered output Clocks, the importer must still call <<fmi3GetClock>> during <<IntermediateUpdateMode>>.]_
```

Comment:
* Or simple "... the importer calls ..."
* Link for out clocks?

Reply:
Done.